executor/builder.go

```diff
@@ -3863,7 +3863,7 @@ func NewRowDecoder(ctx sessionctx.Context, schema *expression.Schema, tbl *model
 	if len(pkCols) == 0 {
 		pkCols = tables.TryGetCommonPkColumnIds(tbl)
 		if len(pkCols) == 0 {
-			pkCols = []int64{0}
+			pkCols = []int64{-1}
```

Comment:
-1 the value before clustered index for this code path...it must be a mistake to change this, so revert to -1

Reply:
related to this bug fix?

Reply:
it seems cannot find a sql to reach here...but maybe we shouldn't change this..- -